doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -0,0 +1,2059 @@
+      "   1286/Unknown - 11s 7ms/step - loss: 1.8623 - accuracy: 0.7538\n",
+      "   1286/Unknown - 11s 7ms/step - loss: 1.8623 - accuracy: 0.7538\n",
+      "   1303/Unknown - 12s 7ms/step - loss: 1.8504 - accuracy: 0.7536\n",
+      "   1303/Unknown - 12s 7ms/step - loss: 1.8504 - accuracy: 0.7536\n",
+      "   1320/Unknown - 12s 7ms/step - loss: 1.8411 - accuracy: 0.7538\n",
+      "   1320/Unknown - 12s 7ms/step - loss: 1.8411 - accuracy: 0.7538\n",
+      "   1338/Unknown - 12s 7ms/step - loss: 1.8234 - accuracy: 0.7547\n",
+      "   1338/Unknown - 12s 7ms/step - loss: 1.8234 - accuracy: 0.7547\n",
+      "   1356/Unknown - 12s 7ms/step - loss: 1.8062 - accuracy: 0.7555\n",
+      "   1356/Unknown - 12s 7ms/step - loss: 1.8062 - accuracy: 0.7555\n",
+      "   1374/Unknown - 12s 7ms/step - loss: 1.7875 - accuracy: 0.7565\n",
+      "   1374/Unknown - 12s 7ms/step - loss: 1.7875 - accuracy: 0.7565\n",
+      "   1391/Unknown - 12s 7ms/step - loss: 1.7748 - accuracy: 0.7572\n",
+      "   1390/Unknown - 12s 7ms/step - loss: 1.7760 - accuracy: 0.7570\n",
+      "1406/1406 [==============================] - 12s 7ms/step - loss: 1.7651 - accuracy: 0.7576\n",
+      "1406/1406 [==============================] - 12s 7ms/step - loss: 1.7651 - accuracy: 0.7576\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-24 14:04:51,032\tERROR checkpoint_manager.py:193 -- Result dict has no key: training_iteration. checkpoint_score_attr must be set to a key of the result dict. Valid keys are ['trial_id', 'experiment_id', 'date', 'timestamp', 'pid', 'hostname', 'node_ip', 'config', 'done']\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Trial TensorflowTrainer_1933d_00000 completed. Last result: \n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2022-05-24 14:04:51,142\tINFO tune.py:753 -- Total run time: 23.11 seconds (22.16 seconds for the tuning loop).\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ray.ml.train.integrations.tensorflow import TensorflowTrainer\n",
+    "\n",
+    "trainer = TensorflowTrainer(\n",
+    "    train_loop_per_worker=train_loop_per_worker,\n",
+    "    scaling_config={\"num_workers\": NUM_WORKERS},\n",
+    "    datasets={\"train\": train_ds},\n",
+    "    preprocessor=get_preprocessor(),\n",
+    ")\n",
+    "result = trainer.fit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Nb0HkOV2R4uL"
+   },
+   "source": [
+    "# Moving on to Serve"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "OlzjlW8QR_q6"
+   },
+   "source": [
+    "Ray Serve serves the trained model through constructs of `ray.serve.model_wrappers.ModelWrapper` and `ray.serve.model_wrappers.ModelWrapperDeployment`. These constructs wrap a `ray.ml.checkpoint.Checkpoint` into an endpoint that can readily serve http requests.\n",
```

Comment:
explain what Ray Serve is, explain what ModelWrapper is, and explain what ModelWrapper Deplooyment is?

You can additionally link to the docstring (and perhaps don't always need to type out the complete path)